### DISCUSSION TITLE
How would I run a test update/install command?

### DISCUSSION THREAD

**Discussion by onyx-nxt** [microsoft/winget-cli#2760] [General]
How would I run a test update/install command? For example, I want to be able to run `winget upgrade APP_NAME` multiple times to test it in my code. How would I do that without modifying the selected app?

**Comment by Trenly** [comment]
This isn’t a feature that currently exists. It is being tracked in:
* #2007 

Adding your thumbs up there will help, as feature requests with the most thumbs up are prioritized over others
